src/libraries/Microsoft.Extensions.Configuration.Binder/src/buildTransitive/EnableConfigurationBindingGenerator.targets

```diff
@@ -1,17 +1,10 @@
 <Project>
-    <Target Name="_Microsoft_Extensions_Configuration_BinderGatherAnalyzers">
```

Comment:
This is per @ViktorHofer's comment in https://github.com/dotnet/runtime/pull/82179/files/f4ade8200bb6efd8f0d73d0cd24100e2ca45ad34#r1135404469.